packages/app/lib/generated_plugin_registrant.dart

```diff
@@ -8,7 +8,6 @@
 
```

Comment:
I think you have to manually delete this file in order for it to be removed from git

Reply:
Yeah I just rush it... My bad ^^